### DISCUSSION TITLE
Angular Build Prod Fails.

### DISCUSSION THREAD

**Discussion by jeetchheda** [community/community#38993] [Codespaces]
Why does codespace fails to Build Angular Apps with --prod flag despite limiting the Node's max_old_space_size to 2GB

**Comment by erikaellr** [comment]
We are happy you are here!

It's hard to tell, based on the information provided, whether this is an issue with Codespaces or something else. You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details and, if possible, a link to a public repo with steps to reproduce the issue, might help someone give you a nudge in the right direction.
